crates/database/src/ethflow_orders.rs

```diff
@@ -116,6 +116,9 @@ pub async fn refundable_orders(
     min_validity_duration: i64,
     min_slippage: f64,
 ) -> Result<Vec<EthOrderPlacement>, sqlx::Error> {
+    /// constraint oq.buy_amount > 0 added to avoid division by zero since table
+    /// order_quotes contains entries with buy_amount = 0 (see
+    /// V021__store_full_quotes.sql)
```

Comment:
Does that mean this this comment is not correct?

Reply:
If you refer to comment from Felix, I added these comments after reading it any my own investigation. There are definitely entries in `order_quotes` with `buy_amount=0`.
Felix commented on ethflow orders not having `buy_amount=0`